triton/apps/vscode.rst

```diff
@@ -136,8 +137,29 @@ Possible issues which may affect usage:
   variables set by Slurm.  Make sure that they don't interfere with
   jobs you may run from this vscode session).
 
-* If you request a GPU node or other high resources, this is reserved
-  the whole time even if you aren't using them.  Consider this before
-  reserving large resources (unless you close the jobs soon), or you
-  might get an email from us asking if we can help you improve
-  research usage.
+Sometimes you want to debug your code using GPU(s). To connect to a
+GPU node, you can follow the above guide, and request a ``gpudev`` partition.
+Note that ``gpudev`` partition has a strict time limitation: Every session
+is maximum 30 minutes long. For more information, see :doc:`partitions 
+<../ref/partitions>`.
```

Comment:
```suggestion
is maximum 30 minutes long. For more information, see :ref:`partitions 
<ref-partitions>`.
```

the sub-pages under ref aren't really designed for direct linking - no headings, etc.  The ref to partitions/index.rst is the main link point.